addon/components/nrg-modal-container/modal-wrapper.hbs

```diff
@@ -0,0 +1,51 @@
+{{! template-lint-disable no-inline-styles }}
+<div
+  class="ui modal visible active
+    {{if @modal.sidebar "sidebar-modal"}}
+    {{if @modal.lightbox "fullscreen lightbox"}}
+    {{if @modal.modalClass @modal.modalClass}}
+    {{if @modal.basic "basic"}}
+    {{if @modal.takeover "side-by-side--takeover"}}"
+  style={{this.modalStyles}}
+  role="dialog"
+  aria-label="Modal"
+  {{did-insert this.modalElementAdded}}
+>
+  {{#if this.hasCloseIcon}}
+    <i class="close icon" role="button" aria-label="Close modal" {{on "click" onHide}}></i>
+  {{/if}}
+  {{#if @modal.headerText}}
+    <div class="header">
+      {{@modal.headerText}}
+    </div>
+  {{/if}}
+
+  <div
+    class="modal-content
+      {{if @modal.lightbox "image"}}
+      {{unless @modal.sidebar "content"}}
+      {{if @modal.scrolling "scrolling"}}"
+    {{did-insert this.addModalToWormhole}}
+    {{will-destroy this.removeModalFromWormhole}}
+  ></div>
+
+  {{#if @modal.hasButtons}}
+    <div class="actions">
+      {{#if @modal.secondaryButton}}
+        <NrgButton
+          @class={{@modal.secondaryButtonClass}}
+          @text={{@modal.secondaryButton}}
+          @onClick={{fn _onSecondary}}
```

Comment:
The `fn` helper is only needed if you want to attach extra arguments to a function like `(fn this.onSelect option)`.

I think this should suffice here:
```suggestion
          @onClick={{this._onSecondary}}
```